### PR TITLE
Fix private-etc of electron-mail, fix geary,minitube

### PR DESCRIPTION
**Pull request by kortewegdevries**
https://imgur.com/nKwR29x

Does someone want to make this a redirect to electron too, it'd be easier to manage that way since most of them are such...?

### CODE REVIEW

**Comment by kortewegdevries** [comment]
Geary has redirect to firefox:

```
Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
```

File: etc/profile-m-z/minitube.profile [hunk 1]
```diff
@@ -46,7 +46,7 @@ notv
 nou2f
 novideo
 protocol unix,inet,inet6,netlink
-seccomp
+seccomp !kcmp
```
Comment (rusty-snake) [inline etc/profile-m-z/minitube.profile:49]: Is this still necessary? Have you set `seccomp-error-action kill`?

Comment (kortewegdevries) [reply etc/profile-m-z/minitube.profile:49]: In `firejail.config`? No. The screen freezes without, on Arch...

Comment (rusty-snake) [reply etc/profile-m-z/minitube.profile:49]: In globals.local or other included overrides.

> The screen freezes without, on Arch...

Ahh, freeze not crash. :+1:

**Review by rusty-snake — APPROVED** [review]

**Comment by rusty-snake** [comment]
Thanks for this contribution!

**Pull request merged**